docs/api/security/Devicekey.md

```diff
@@ -41,6 +45,12 @@ To instantiate DeviceKey, you need to call its `get_instance` member function as
     DeviceKey &deviceKey = DeviceKey::get_instance();
 ```
 
+To generate root of trust before key derivation:
+
+```c++ TODO
+    DeviceKey &deviceKey = DeviceKey::get_instance().generate_root_of_trust();
```

Comment:
This code example does not seem right. We don't get the reference as a return value, but an error code instead. Maybe you meant:
```C++

int err = DeviceKey::get_instance().generate_root_of_trust();
if (err == 0)
    // Success
```